controllers/machinedeletionremediation_controller.go

```diff
@@ -38,9 +38,10 @@ import (
 )
 
 const (
-	machineAnnotationOpenshift = "machine.openshift.io/machine"
-	machineKind                = "Machine"
-	machineSetKind             = "MachineSet"
+	machineAnnotationOpenshift     = "machine.openshift.io/machine"
+	machineKind                    = "Machine"
+	machineSetKind                 = "MachineSet"
+	MachineNameNamespaceAnnotation = "nodehealthcheck.medik8s.io/machineNameNamespaceAnnotation"
```

Comment:
Why nodehealthcheck? It's MDR's annotation 🙂

Reply:
also, value can be without "Annotation" IMHO... it's clear from where it's set

Reply:
I agree on both